src/app/components/SearchForm/SearchForm.jsx

```diff
@@ -121,8 +122,37 @@ class SearchForm extends React.Component {
     const selectedField = this.state.searchQuery.showField || this.state.searchQuery.queries[0].field;
 
     return (
-      <div className="grid-row">
-        <form
+      <DS.SearchPromo
+        headingText="Search the World's Research Collections"
```

Comment:
This whole series of props looks very hard to read to me, and I'm wondering if there's a better way to order/sort/put them in `{...inputprops}, {...searchBarProps}, {...paragraphProps}` or something.  

@EdwinGuzman @helenvholmes does this bother you as well? Thoughts?

Reply:
I prefer one prop per line. I'd also make the `advancedSearchMessage` value into a variable. If it's easier for _you_ to combine into a variable and explode it, then go for it.

(Some package conventions like prettier opt for one prop per line, too).